src/ensembl/src/content/app/entity-viewer/gene-view/components/transcripts-filter/TranscriptsFilter.scss

```diff
@@ -2,44 +2,35 @@
 @import 'src/content/app/entity-viewer/gene-view/styles/constants';
 
 .container {
-  display: grid;
-  grid-template-columns: $left_column $middle_column_left_gap auto;
-  padding: 9px 0;
-}
-
-.filterLabel {
-  grid-column: 1/2;
-  text-align: right;
-  padding-right: 2px;
-  cursor: pointer;
-}
-
-.chevron {
-  margin-left: 10px;
-  height: 12px;
-  width: 12px;
-  margin-bottom: -3px;
+  grid-area: 3/1/3/4;  
+  position: sticky;
+  height: 100%;
+  top: -5px;
+  z-index: 3;  
 }
 
 .filterBox {
   display: grid;
-  grid-column: 3;
-  grid-template-columns: 270px minmax(210px, auto) 20px;
+  grid-column: 4;
+  grid-template-columns: 160px 270px minmax(210px, auto) 20px;
   background-color: $soft-black;
   color: $white;
   text-align: left;
   padding: 15px 15px 45px 20px;
-  margin-top: 2px;
-  margin-left: 2px;
   font-weight: 300;
+  border-bottom-left-radius: 3px;
+  border-bottom-right-radius: 3px;
+  box-shadow: -1px 3px 9px 2px rgba(0, 0, 0, 0.5);
 
-  @media screen and (min-width: 1600px) {
-    width: calc(100% - 320px);
+  // Keeping this here as we might need to revise this fix when
+  // we have the horizontal scrollbar
```

Comment:
This is our minimum width case: 980px:

![reposition-filter-box review ensembl org_entity-viewer_homo_sapiens_GCA_000001405_28_gene_ENSG00000109339_view=transcripts](https://user-images.githubusercontent.com/6834224/120823100-f98fbd80-c54e-11eb-8d28-316b366e6928.png)

Were you planning an independent horizontal scrollbar in the filter box?

Reply:
Not on the filter box.

Reply:
Then why keep the commented out code? We usually don't keep the commented code; we just remove it trusting our version control system 🙂

Reply:
That code is to calculate the width of the filter box on small screen, can remember exactly the purpose but it was to show the box with the right gap on small screen. And i remember it took us quite a few calculations to get it right so thought just in case we might need again and we dont have to go through the pain of getting the numbers right again, left it there in case it might help when we are doing the horizontal bar.

Reply:
The code just says: take 100% of the available width and subtract the width that the opened sidebar would add. We can easily recreate it if necessary. If Andrea is fine with the close button on the filter panel getting behind the sidebar, we can safely delete the old code.

![image](https://user-images.githubusercontent.com/6834224/120841445-690fa800-c563-11eb-8603-6907f4e75988.png)

Reply:
Thanks for this, it is indeed what we wanted to do :) I will check with Andrea because for smaller screen (even laptop screen) the cross is not visible which i think andrea insisted on. I will check with her.

Reply:
Done (remove commented code)